syntax/parser.go

```diff
@@ -2046,7 +2046,8 @@ func (p *Parser) wordIter(ftok string, fpos Pos) *WordIter {
 		return wi
 	}
 	p.got(_Newl)
-	if pos, ok := p.gotRsrv("in"); ok {
+	switch pos, ok := p.gotRsrv("in"); {
```

Comment:
Like in the previous case, I think using a define-switch here is overkill for just three cases.

Reply:
Done in e336613e7883ab4a30afe31f6d84c28752fa2f75.